standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/txn/TxnHandler.java

```diff
@@ -377,6 +377,8 @@ public void setConf(Configuration conf){
     numOpenTxns = Metrics.getOrCreateGauge(MetricsConstants.NUM_OPEN_TXNS);
 
     timeout = MetastoreConf.getTimeVar(conf, ConfVars.TXN_TIMEOUT, TimeUnit.MILLISECONDS);
+    replicationTxnTimeout = MetastoreConf.getTimeVar(conf, ConfVars.REPL_EVENT_DB_LISTENER_TTL, TimeUnit.MILLISECONDS) +
```

Comment:
REPL_EVENT_DB_LISTENER_TTL value should be picked from source cluster. If the value is different in source and target cluster, we can't automatically determine this. Instead we should update the timeout config description and provide this as a guideline.